lesson_10/lesson-10-rusu-daniel/Tema_polygon.py

```diff
@@ -0,0 +1,61 @@
+import math
+
+class Mixxin:
+    def perim(self): 
+        return sum(self.sides)
+
+class Polygon:
+
+    def __init__(self, *args):
+        self.sides = list(args)
+
+    def __str__(self):
+        return f'Polygon object has {list(self.sides)} sides'
+
+    def display(self):
+        for index, side in enumerate(self.sides, start=1):
+            print(f'Side {index} with length: {side}')
+            
+class PerimObject(Mixxin,Polygon):
```

Comment:
aici in loc de clasa PerimObject puteai sa adaugi `Mixxin` direct ca parinte al clasei Square.